.buildkite/publish/build-multiarch-docker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -exu
+set -o pipefail
+
+function realpath {
+  echo "$(cd "$(dirname "$1")"; pwd)"/"$(basename "$1")";
+}
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+BUILDKITE_DIR=$(realpath "$(dirname "$SCRIPT_DIR")")
+PROJECT_ROOT=$(realpath "$(dirname "$BUILDKITE_DIR")")
+
+VERSION_PATH="$PROJECT_ROOT/connectors/VERSION"
+VERSION=$(cat $VERSION_PATH)
```

Comment:
worth extracting this into one file that can be `source` d from multiple places, so that we don't have to maintain it in multiple places?

Reply:
Yes! I was considering that...